zutil.h

```diff
@@ -120,6 +120,10 @@ extern z_const char * const PREFIX(z_errmsg)[10]; /* indexed by 2-zlib_error */
 #  define OS_CODE  3  /* assume Unix */
 #endif
 
+         /* macros */
+
+#define CHECK_VER_STSIZE(_ver,_stsize) ((_ver) == NULL || (_ver)[0] != PREFIX2(VERSION)[0] || (_stsize) != (int32_t)sizeof(PREFIX3(stream)))
```

Comment:
No need for variables to be prefixed with underscore in macro.

Reply:
Can't remember why I did that, but I think it had to have been some warning, not sure.

Reply:
Like with any preprocessor token, a macro parameter name might shadow something that is previously defined or declared at the point of macro invocation... 

That is why sometimes, for disambiguation, tokens are prefixed and/or surrounded by parentheses to avoid accidental substitution, or concatenation of two unrelated tokens.

Preprocessor tokens can also match other tokens partially. This is because preprocessor doesn't require whitespace as separator of tokens, any invalid character (not allowed in token name) will separate two tokens.